build.sbt

```diff
@@ -68,6 +65,12 @@ lazy val integrationTestSettings = Seq(
   concurrentRestrictions in Test := Seq(Tags.limitAll(math.max(1, java.lang.Runtime.getRuntime.availableProcessors() / 2)))
 )
 
+val pbSettings = ProtobufPlugin.projectSettings ++ Seq(
```

Comment:
Could you add some doc? It would make it a little easier to maintain.

Reply:
Done